server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -364,6 +376,14 @@ export class AgenticChatController implements ChatHandlers {
         return this.#tabBarController.onConversationClick(params)
     }
 
+    async onListMcpServers(params: ListMcpServersParams) {
+        return this.#mcpEventHandler.onListMcpServers(params)
+    }
+
+    async onMcpServerClick(params: McpServerClickParams) {
+        return this.#mcpEventHandler.onMcpServerClick(params)
+    }
```

Comment:
Should these functions have an inner check if the clientcapabilities support mcp, and do nothing if there is no support?